testing/side-by-side.tsx

```diff
@@ -4,34 +4,32 @@ import {HeadingMedium} from "@khanacademy/wonder-blocks-typography";
 import * as React from "react";
 import ReactJson from "react-json-view";
 
-import {interactiveSizes} from "../packages/perseus/src/styles/constants";
-
 type Props = {
-    leftTitle: React.ReactNode;
-    left: React.ReactNode;
-    rightTitle?: string;
+    rendererTitle: React.ReactNode;
+    renderer: React.ReactNode;
+    JSONTitle?: string;
     jsonObject: any;
 };
 
-const SideBySide = ({
-    leftTitle = "Renderer",
-    left,
-    rightTitle = "Perseus JSON",
+const SplitView = ({
+    rendererTitle = "Renderer",
+    renderer,
+    JSONTitle = "Perseus JSON",
     jsonObject,
 }: Props): React.ReactElement => {
     return (
         <View style={styles.sideBySide}>
-            <View style={styles.leftPanel} className="framework-perseus">
-                <HeadingMedium>{leftTitle}</HeadingMedium>
-                {left}
+            <View className="framework-perseus">
+                <HeadingMedium>{rendererTitle}</HeadingMedium>
+                {renderer}
             </View>
-            <View style={styles.rightPanel}>
-                <HeadingMedium>{rightTitle}</HeadingMedium>
+            <View>
+                <HeadingMedium>{JSONTitle}</HeadingMedium>
                 <ReactJson
                     style={{marginTop: "10px"}}
                     quotesOnKeys={false}
                     enableClipboard={false}
-                    collapsed={5}
+                    collapsed={true}
```

Comment:
This auto-collapses the PerseusJSON view so that it doesn't fill the page, and push the Storybook Controls "below the fold".